Src/BootCamp.Chapter/LogDemo.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    internal class LogDemo
+    {
+        public static void Demo(Ilogger logger)
+        {
+            Console.ForegroundColor = ConsoleColor.Green;
+            logger.Log($"INFO  {DateTime.Now} The programm has started {Environment.NewLine}");
+            Console.ForegroundColor = ConsoleColor.White;
+
+            for (int i = 0; i < 2; i++)
+            {
+                float weight, height;
+                GetPersonData(logger, out weight, out height);
+                CalculateBMI(logger, weight, height);
+
+            }
+
+            Console.ForegroundColor = ConsoleColor.Green;
+            logger.Log($"INFO  {DateTime.Now} The programm has stoppped {Environment.NewLine}");
+            Console.ForegroundColor = ConsoleColor.White;
+        }
+
+        private static void CalculateBMI(Ilogger logger, float weight, float height)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename method 'CalculateBMI' to match pascal case naming rules, consider using 'CalculateBmi'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5137406)